test/generator/string_param_generator.cpp

```diff
@@ -0,0 +1,24 @@
+#include "Halide.h"
```

Comment:
The test needs to be augmented to ensure that setting the generatorparam in the Makefile (and also the CMake file) works correctly.

Reply:
We also need a test to verify that GeneratorParam<string> works correctly with Stubs; the easiest way to do that is add one to stubtest_generator.cpp and then to reference it in stubtest_aottest.cpp and also in stubuser_generator.cpp